create_gh_mirrors.py

```diff
@@ -163,11 +163,16 @@ def main():
     html_tpl_path = 'upload/landing_gh.tpl'
     github_access_token = ''
 
-    tb_version_config = ConfigParser.ConfigParser()
-    tb_version_config.read(tb_version_path)
-    tb_version = tb_version_config.get('version', 'current')
+    try:
+        tb_version_config = configparser.ConfigParser()
+        tb_version_config.read(tb_version_path)
+        tb_version = tb_version_config['version']['current']
+
+    except:
```

Comment:
Let's not do an unconditional `except:` -- instead, let's catch specific exceptions.